postgres/tests/test_pg_integration.py

```diff
@@ -217,6 +217,31 @@ def test_config_tags_is_unchanged_between_checks(integration_check, pg_instance)
         assert check._config.tags == expected_tags
 
 
+def test_correct_hostname(aggregator, integration_check, pg_instance):
+    pg_instance['dbm'] = True
```

Comment:
May be good to explicitly test that hostname is not set when `dbm` is not enabled